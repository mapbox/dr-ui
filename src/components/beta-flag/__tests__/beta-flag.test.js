import React from 'react';
import renderer from 'react-test-renderer';
import { testCases } from './beta-flag-test-cases.js';

describe('back-top-top-button', () => {
  describe(testCases.basic.description, () => {
    let testCase;
    let wrapper;
    let tree;

    beforeEach(() => {
      testCase = testCases.basic;
      wrapper = renderer.create(
        React.createElement(testCase.component, testCase.props)
      );
      tree = wrapper.toJSON();
    });

    test('renders as expected', () => {
      expect(tree).toMatchSnapshot();
    });
  });
});
