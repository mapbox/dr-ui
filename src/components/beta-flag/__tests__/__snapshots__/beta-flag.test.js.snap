// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`back-top-top-button Basic renders as expected 1`] = `
<div
  className="inline-block"
  onBlur={[Function]}
  onClick={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
>
  <div
    aria-describedby="tooltip-1"
    className="txt-s txt-bold round px6 inline-block cursor-default border border--green-light"
    style={
      Object {
        "backgroundColor": "rgb(232, 245, 238)",
        "color": "rgb(27, 125, 79)",
      }
    }
  >
    Beta
  </div>
  <div
    className="hide-visually"
    id="tooltip-1"
    role="tooltip"
  >
    <div
      className="txt-s wmax120"
    >
      This feature is in public beta and is subject to changes.
    </div>
  </div>
</div>
`;
